diesel/src/pg/backend.rs

```diff
@@ -138,18 +138,18 @@ impl SqlDialect for Pg {
     type SelectStatementSyntax = sql_dialect::select_statement_syntax::AnsiSqlSelectStatement;
 
     type ExistsSyntax = sql_dialect::exists_syntax::AnsiSqlExistsSyntax;
-    type ArrayComparison = PgStyleArrayComparision;
+    type ArrayComparison = PgStyleArrayComparison;
 }
 
 impl DieselReserveSpecialization for Pg {}
 impl TrustedBackend for Pg {}
 
 #[derive(Debug, Copy, Clone)]
-pub struct PgOnConflictClaues;
+pub struct PgOnConflictClause;
```

Comment:
I'm not sure if it's a public API change, if so, we need to inform people about it :)

Reply:
Yes that type is not part of the public API, so it should be fine to change it.